### ISSUE TITLE
Add `<series id>` to tags

### ISSUE THREAD

**Issue by CLHatch** [rmcrackan/Libation#744]
I was thinking it would be nice to be able to include the series ASIN in the naming pattern, so we could have something like this folder structure:

`The Age of Madness [B07YDZ1Q4R]\01 - (2019) - A Little Hatred꞉ The Age of Madness. Book 1 [1478916591]`

A tagname that comes to mind is `<series id>`, to go along with the existing `<id>` tag.

Edit:
I haven't looked at the metadata retrieved to see if the "series" and "podcast" id are the same, but if not, it might make sense to have a `<podcast id>` as well.  Either that, or have `<series id>` use whichever is present, since a single "book" won't be both in a "series" and a "podcast" anyways.

**Comment by ElysiumFic** [comment]
I agree with this. Also, <author id> might not be a terrible idea, either.
Here's my ideal setup, so that no matter how or where or what software I try to use with my audiobooks, I've got everything I need at my fingertips with minimal effort.

/<first author> [<author id>]/<if series><series> [<series id>]/<series part>. <-if series><title short> - <year> {<first narrator>}/
<title> (<year>).m4b
<title> (<year>).cue
<title> (<year>).jpg
<title> (<year>).metadata.json

Then also include options for metadata.nfo, metadata.opf, desc.txt, reader.txt, or whatever else the current popular systems of audiobook organization tend to call for.

And then, also include an option for integrating externally acquired (non-Audible) audiobooks into this structure, including tagging them with Audible metadata.

**Comment by CLHatch** [comment]
> I agree with this. Also, might not be a terrible idea, either. Here's my ideal setup, so that no matter how or where or what software I try to use with my audiobooks, I've got everything I need at my fingertips with minimal effort.
> 
> ```
> /<first author> [<author id>]/<if series><series> [<series id>]/<series part>. <-if series><title short> - <year> {<first narrator>}/
> <title> (<year>).m4b
> <title> (<year>).cue
> <title> (<year>).jpg
> <title> (<year>).metadata.json
> ```
> 
> Then also include options for metadata.nfo, metadata.opf, desc.txt, reader.txt, or whatever else the current popular systems of audiobook organization tend to call for.
> 
> And then, also include an option for integrating externally acquired (non-Audible) audiobooks into this structure, including tagging them with Audible metadata.

This is my current naming scheme (sort of tailored to Audiobookshelf):

Folder:
`<if podcast->Podcasts\Audible\<account nickname>\<author[separator(; )]>\<if series-><series>\<series#[00.##]> - <-if series>(<pub date>) - <series> - <title> [<id>]<-if podcast><!if podcast->AudioBooks\Audible\<account nickname>\<author[separator(; )]>\<if series-><series>\<series#[00.##]> - <-if series>(<year>) - <title> [<id>]<-if podcast>`

File:
`<audible title> [<id>]`

An example file path for this is:
`C:\Users\CLHat\Libation\Books\AudioBooks\Audible\Chris\Tim F. LaHaye; Jerry B. Jenkins\Left Behind\13 - (2007) - Kingdom Come꞉ The Final Victory [B002V5BGAA]\Kingdom Come [B002V5BGAA].m4b`

I used to add the narrator(s) also, but the paths were getting too long, and with the ASIN included it will pick that info up anyways.  I have it create the other files you do as well.  No need to try to have Audible create other program's metadata files, really.  Too likely the format of those will change later.

**Comment by rmcrackan** [comment]
@ElysiumFic -- good point. I also don't see them.

@Mbucari -- Was a check-in missed during #1223 ?

**Comment by CLHatch** [comment]
> [@ElysiumFic](https://github.com/ElysiumFic) -- good point. I also don't see them.
> 
> [@Mbucari](https://github.com/Mbucari) -- Was a check-in missed during [#1223](https://github.com/rmcrackan/Libation/pull/1223) ?

I thought they were missing at first too. But instead of doing something like `<series id>` he added a `format(id)`.

**Comment by rmcrackan** [comment]
@CLHatch That's right, thanks for the reminder

@ElysiumFic Look in [Tag Formatters](https://github.com/rmcrackan/Libation/blob/master/Documentation/NamingTemplates.md#tag-formatters). `ID` is one of the new options.

@Mbucari Ignore me. Thanks for keeping tests and doc.s up to date.
